packages/noya-state/src/reducers/application.ts

```diff
@@ -73,12 +77,17 @@ export type Action =
   | [type: 'highlightLayer', highlight: LayerHighlight | undefined]
   | [type: 'setExpandedInLayerList', layerId: string, expanded: boolean]
   | [type: 'selectPage', pageId: UUID]
-  | [type: 'alignLeft']
-  | [type: 'alignCenterHorizontally']
-  | [type: 'alignRight']
-  | [type: 'alignTop']
-  | [type: 'alignCenterVertically']
-  | [type: 'alignBottom']
+  | [type: 'distribute', placement: 'horizontal' | 'vertical']
```

Comment:
Can we rename the actions to `distributeLayers` and `alignLayers`?

I'd like most action names to follow the format `verbObject`, like `selectPage` or `highlightLayer`. Some of them probably don't right now... but now that they're growing a bit, I'll try to keep them mostly to that format

Reply:
Oops yes, not enough info there 😅  fixed!